code/Model/EditableFormField/EditableOption.php

```diff
@@ -66,12 +66,12 @@ public static function set_allow_empty_values($allow)
     }
 
     /**
-     * @deprecated 5.0 Use "$Title.XML" in templates instead
+     * @deprecated 5.0..6.0 Use "$Title" in templates instead
      * @return string
      */
     public function getEscapedTitle()
     {
-        return Convert::raw2att($this->Title);
+        return $this->Title;
```

Comment:
Does this not result in a change of behaviour for this method? I think it should probably remain escaped but the casting helper can cast it as HTML rather than text so we don't get double escaping if it's ever used in a template.

The other changes should then use `Title` as you have done

Reply:
Fair call, I've reverted the behaviour of this method - the rest of the changes are separate and won't be affected by this